### PR TITLE
Update lede common irving app to adhere to (nearly) unmodified airbnb standards

### PR DESCRIPTION
**Pull request by ostowe**
## Issue(s): Relates to or closes...
Yes, but ticket TBD

## Summary: This pull request will...
* Update eslint standards for all irving core packages to adhere (nearly) 100% to the unmodified airbnb config.

## Tests: I know this code works because...
Tested locally with lede.

### CODE REVIEW

File: packages/wordpress/components/blockStyles/blocks/heading.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { css } from 'styled-components';
 import { link } from './utils';
 
-/* eslint-disable indent */
+/* eslint-disable indent, import/prefer-default-export */
```
Comment (justinwhall) [inline packages/wordpress/components/blockStyles/blocks/heading.js:4]: I wonder if it's worth adding an `.eslint` file in `blockStyles` if this is a common rule(s) we are disabling.

```
{
    "rules": {
        "import/prefer-default-export": "off"
    }
}
```

**Review by justinwhall — APPROVED** [review]
🚢

File: packages/wordpress/components/blockStyles/blocks/heading.js (hunk 1 above)
Comment (ostowe) [reply packages/wordpress/components/blockStyles/blocks/heading.js:4]: hmmmm true, that might be helpful. this is pretty common throughout the lede theme too...maybe we should just disable across the board within lede?

Comment (ostowe) [reply packages/wordpress/components/blockStyles/blocks/heading.js:4]: Oh wait, I already disabled it in irving core but didn't remove these eslint disable lines.

**Pull request merged**